app/controllers/devise/cas_sessions_controller.rb

```diff
@@ -5,7 +5,11 @@ class Devise::CasSessionsController < Devise::SessionsController
     unloadable
   end
 
-  skip_before_filter :verify_authenticity_token, :only => [:single_sign_out], :raise => false
+  if Rails.version =~ /^5/
```

Comment:
Similar to #136, we'll probably need to change this to `if Rails::VERSION::MAJOR >= 5` to avoid weird errors.